frontend/src/pages/Sessions/IntegrationCard/IntegrationCard.tsx

```diff
@@ -1,35 +1,25 @@
-import DemoWorkspaceButton, {
-	DEMO_WORKSPACE_APPLICATION_ID,
-	DEMO_WORKSPACE_PROXY_APPLICATION_ID,
-} from '@components/DemoWorkspaceButton/DemoWorkspaceButton'
-import { useParams } from '@util/react-router/useParams'
+import DemoWorkspaceButton from '@components/DemoWorkspaceButton/DemoWorkspaceButton'
+import FullBleedCard from '@components/FullBleedCard/FullBleedCard'
+import { useProjectId } from '@hooks/useProjectId'
 import Lottie from 'lottie-react'
 import React from 'react'
 import { Link } from 'react-router-dom'
 
-import FullBleedCard from '../../../components/FullBleedCard/FullBleedCard'
 import WaitingAnimation from '../../../lottie/waiting.json'
 import styles from './IntegrationCard.module.scss'
 
 export const IntegrationCard = () => {
-	const { project_id } = useParams<{ project_id: string }>()
-	const projectIdRemapped =
-		project_id === DEMO_WORKSPACE_APPLICATION_ID
-			? DEMO_WORKSPACE_PROXY_APPLICATION_ID
-			: project_id
```

Comment:
Is this fix needed? Wouldn't removing this remove the demo project?

Reply:
We do the remapping inside the useProjectId hook:

https://github.com/highlight/highlight/blob/fix/3733/frontend/src/hooks/useProjectId.ts#L12